newsroom/oauth_clients/clients_async.py

```diff
@@ -0,0 +1,47 @@
+from content_api import MONGO_PREFIX
+from typing import Optional, Annotated, List, Dict, Union
+from superdesk.core.resources import ResourceModel, ResourceConfig, MongoResourceConfig
+from superdesk.core.resources.service import AsyncResourceService
+from superdesk.core.web import EndpointGroup
+from pydantic import Field
+import logging
+from bson import ObjectId
+
+
+class ClientResource(ResourceModel):
+    id: Annotated[Union[str, ObjectId], Field(alias="_id")] = None
+    name: str
+    password: str
+    last_active: Optional[str] = None
+    etag: Annotated[Optional[str], Field(alias="_etag")] = None
```

Comment:
We're not currently using `_etag` with new code. This will be done in a later ticket (and added to the `ResourceModel`)